google-cloud-firestore/lib/google/cloud/firestore/query.rb

```diff
@@ -1216,7 +1222,7 @@ def doc_id_path
         ##
         # @private Raise an error unless an database available.
         def ensure_client!
-          raise "Must have active connection to service" unless client
+          raise "Must have active connection to client" unless client
```

Comment:
I would think the original word "service" made more sense. You "connect to" a service, not to a client.

Reply:
Agreed, I was looking at it too narrowly. Restored.